released/discovery_center/mission_4327/setup_subaccount_cf/variables.tf

```diff
@@ -0,0 +1,198 @@
+######################################################################
+# Customer account setup
+######################################################################
+# subaccount
+variable "globalaccount" {
+  type        = string
+  description = "The globalaccount subdomain."
+  default     = "yourglobalaccount"
+}
+# subaccount
+variable "subaccount_name" {
+  type        = string
+  description = "The subaccount name."
+  default     = "UC - Events to Business Actions"
+}
+# Region
+variable "region" {
+  type        = string
+  description = "The region where the project account shall be created in."
+  default     = "us10"
+}
+# Cloudfoundry environment label
+variable "cf_environment_label" {
+  type        = string
+  description = "The Cloudfoundry environment label"
+  default     = "cf-us10"
+}
+
+# Cloudfoundry space name
+variable "cf_space_name" {
+  type        = string
+  description = "The Cloudfoundry space name"
+  default     = "dev"
+}
+
+# hana password
+variable "hana_cloud_system_password" {
```

Comment:
Here you should add sensitive=true
I also would not put a default for sensitive values